### ISSUE TITLE
Looking for Arabic translation of Synfig UI

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#1198]
Hi everyone!
I am looking for someone who can translate Synfig UI into Arabic. Anyone interested? Any ideas where to look for contributors?

**Comment by OBKF** [comment]
I will do it, I am a bit busy so it may take some time.
Please link me the files.

**Comment by morevnaproject** [comment]
Hi @OBKF! Great, thank you!
We are using using Transifex service, so you can translate online using web browser. So just sign up here - https://www.transifex.com/morevnaproject/synfig/
Please let me know if that's okay fro you. __^

**Comment by mhamadharmush** [comment]
I will do it insha'allah

**Comment by OBKF** [comment]
@morevnaproject Yeah, Nice. I will start as soon as I can finish some urgent work I am doing.

@mhamadharmush Great, we will help each other making this translation perfect.

**Comment by morevnaproject** [comment]
Thanks everyone! ^__^

**Comment by boubouh** [comment]
Good work gays ✌️💪

**Comment by morevnaproject** [comment]
Much thanks to everyone who contributing to Arabic translations! First results of your work are already available in latest development release, Synfig Studio 1.3.13! - https://www.synfig.org/2020/03/17/synfig-studio-1-3-13-released/ ^__^

![screenshot_004](https://user-images.githubusercontent.com/332868/76853496-f6dc7600-687f-11ea-930d-9de1888ec129.png)

:tada:

**Comment by morevnaproject** [comment]
Much thanks to everyone who contributed to translation! The translation is 72% complete now and it is shipped with Synfig 1.3.14 - https://www.synfig.org/2020/05/07/synfig-studio-1-3-14-released/ ^__^

![screenshot_002](https://user-images.githubusercontent.com/332868/81320267-1c853f00-90bb-11ea-8809-93d5dabf99a2.png)

**Comment by morevnaproject** [comment]
So, the process got stuck at 74%. Anyone can help with final push? ^__^

**Comment by morevnaproject** [comment]
Still looking for someone who can help finish Arabic translations. Your help is much appreciated! ^__^

**Comment by OBKF** [comment]
@morevnaproject I am sorry I couldn't help, my PC died and I had to  work on a lot of stuff during this time.

I hope I can do something as soon as possible.

**Comment by morevnaproject** [comment]
Thank you!

**Comment by mohamedAdhamc** [comment]
> Still looking for someone who can help finish Arabic translations. Your help is much appreciated! ^__^

Hey if this issue is still not complete I believe I may be able to help. I have been a contributor here for a bit and I'm also bilingual due to being raised in the U.S and then moving back to Egypt. Arabic is my first language (although not standard Arabic but no Arabian countries speak standard Arabic anymore, even though it's still the official way of typing and communicating).

**Comment by ice0** [comment]
@mohamedAdhamc 
Awesome! You can check the translation progress here:
https://www.transifex.com/morevnaproject/synfig/

**Comment by mohamedAdhamc** [comment]
> @mohamedAdhamc Awesome! You can check the translation progress here: https://www.transifex.com/morevnaproject/synfig/

Great will be starting right when I wake up tomorrow hopefully 😄.

**Comment by mohamedAdhamc** [comment]
request to join the arabic team sent.
